rules/windows/discovery_posh_suspicious_api_functions.toml

```diff
@@ -20,30 +20,36 @@ note = """## Triage and analysis.
 
 ### Investigating PowerShell Suspicious Discovery Related Windows API Functions
 
-PowerShell is one of the main tools used by system administrators for automation, report routines, and other tasks.
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making
+it available for use in various environments, creating an attractive way for attackers to execute code.
 
-Attackers can use PowerShell to interact with the Win32 API to bypass file based antivirus detections, using libraries
+Attackers can use PowerShell to interact with the Win32 API to bypass command line based detections, using libraries
 like PSReflect or Get-ProcAddress Cmdlet.
 
-#### Possible investigation steps:
+#### Possible investigation steps
 
 - Examine script content that triggered the detection. 
-- Investigate script execution chain (parent process tree).
+- Investigate the script execution chain (parent process tree).
 - Inspect any file or network events from the suspicious PowerShell host process instance.
-- If the action is suspicious for the user, check for any other activities done by the user in the last 48 hours.
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Check whether it makes sense for the user to use PowerShell to complete its tasks.
+- Check if the imported function was executed.
 
-### False Positive Analysis
+### False positive analysis
 
-- Verify whether the script content is malicious/harmful.
+- Discovery activities themselves are not inherently malicious if occurring in isolation, as long as the script does not
+contain other capabilities, and there are no other alerts related to the user or host; such alerts can be dismissed.
```

Comment:
```suggestion
contain other capabilities. Also, if no other alerts are related to the user or host, you can dismiss false positive alerts. 
```

Reply:
Not sure if the second sentence still works but if you can create a new sentence that's separate from the first one, it might improve readability.